go.mod

```diff
@@ -1 +1,5 @@
 module github.com/getlantern/uuid
+
+go 1.12
```

Comment:
Do we really need this to require 1.12? That will likely cause issues with our current client side builds that require 1.10 for XP and earlier OSX.

Reply:
Oh, because Go 1.10 is the last release to support Windows XP, huh? Didn't realize that. Yeah, this line isn't necessary, just removed it